src/pages/code-of-conduct.tsx

```diff
@@ -0,0 +1,33 @@
+import Head from "next/head"
+import React from "react"
+import styles from "@/styles/page.module.css"
+import { TextBlock, CodeOfConductText } from "@/components"
+import Link from 'next/link';
+
+export default function CodeOfConduct() {
+  return (
+    <>
+
+      <Head>
+        <title>Empathetech Code of Conduct</title>
+        <meta name="description" content="code of conduct for empathetech" />
```

Comment:
Let's capitalize "Code" here in the content description

Reply:
```
<meta name="description" content="Code of conduct for empathetech" />
```
Like so? Should Empathetech be caps too?